### DISCUSSION TITLE
Qualifier Explanations added to documentation

### DISCUSSION THREAD

**Discussion by walinchus** [USEPA/RAQSAPI#21] [Q&A] [answered]
Each observation contains a list of one of 39 qualifiers. However, the documentation doesn't include what each of these qualifiers means and I can't find anything about it on the web: 
<img width="737" alt="image" src="https://github.com/USEPA/RAQSAPI/assets/25959269/3c15e454-a5c7-4e3e-81b7-f31123b0e558">

For example, why are some values voided by the operator. One qualifier just says "outlier." Does that mean the machine was malfunctioning or that's just a reading outside of the norm? If the machine was malfunctioning why not label it "machine malfunction?" Etc. 

Thank you for your work on this package.

**Comment by mccroweyclinton-EPA** [comment, marked as answer]
Thank you @walinchus for your interests in RAQSAPI.

The AQS DataMart API is simply a portal for public access to air quality data submitted to AQS. The vast majority of air monitoring data that is submitted to AQS originates from state, local, tribal, and industrial organizations that are not a part of the EPA. The data that is in AQS is the property of the agency that is responsible for uploading the data to AQS. AQS in of itself does not apply quality assurance qualifier codes on the data that is submitted. You will need to refer to the air monitoring agency that is responsible for the particular measurements in question to find out as to why they chose to apply the qualifiers to the data in question.

For a list of all of the qualifier codes that are available please refer to the [Qualifiers list](https://aqs.epa.gov/aqsweb/documents/codetables/qualifiers.html) under the [AQS Code list page](https://www.epa.gov/aqs/aqs-code-list).

For an overview of the structure and content of the AQS system please refer to [About AQS Data](https://aqs.epa.gov/aqsweb/documents/about_aqs_data.html#interpreting-aqs-data) which is referenced in the RAQSAPI readme. Section 7 discusses the ownership of the data.

**Reply by walinchus** [reply]
Okay great thanks.
